lib/morse_translator.rb

```diff
@@ -40,4 +41,12 @@ def initialize
                     " " => " "}
   end
 
+  def translate(english)
+    in_string = english.downcase.chars
+    out_string = []
+    in_string.map do |letter|
```

Comment:
If you're going to use map, don't need the `out_string` aggregator. That pattern is intended to be used with `.each`, when you either don't have access to `map` (for example in a language other than Ruby), or aren't aware of a more specific enumerable.

That said, like that you were able to get this to work!

Reply:
This one was a time issue